src/analytics/Properties.tsx

```diff
@@ -1641,19 +1641,28 @@ interface EarnEventsProperties {
   [EarnEvents.earn_active_pools_card_press]: undefined
   [EarnEvents.earn_home_learn_more_press]: undefined
   [EarnEvents.earn_pool_card_press]: {
-    tokenAmount: string
+    poolAmount: string
     poolId: string
```

Comment:
can we add poolId to EarnCommonProperties and enforce it for every pool specific event?